2.13/tests/test/test.js

```diff
@@ -330,9 +295,21 @@ function testUpdate(instanceId, validBody, isAsync){
                 if (err) return done(err);
                 var message = validateJsonSchema(res.body, updateResponseSchema);
                 if (message!='') done(new Error(message));
-                testLastOperationStatus(res.body, done)
+                else done();
             })
         });
+
+        if (isAsync) {
+            // TODO: the query string should contain 'operation' for the last operation. FYI: https://github.com/openservicebrokerapi/servicebroker/blob/v2.13/spec.md#parameters.
+            describe('UPDATE - poll', function() {
+                testAPIVersionHeader('/v2/service_instances/' + instanceId + '/last_operation', 'GET');
+                testAuthentication('/v2/service_instances/' + instanceId + '/last_operation', 'GET');

```

Comment:
Not need to test version header and authentication again here. This path is already tested in `describe('PROVISION - poll')`

Reply:
Please help remove the duplicate test, thanks!